frontend/model/captureLogs.js

```diff
@@ -0,0 +1,186 @@
+import sbp from '~/shared/sbp.js'
+import { CAPTURED_LOGS, SET_APP_LOGS_FILTER } from '~/frontend/utils/events.js'
+
+/*
+  giConsole/count - total logs stored
+  giConsole/lastEntry - latest log hash.
+  giConsole/markers - an array of markers at every 1000th entry.
+*/
+const ENTRIES_LIMIT = 4000
+const ENTRIES_MARKER = 1000
+
+let isCapturing = true
+let isConsoleOveride = false
+let lastEntry = null
+let entriesCount = null
```

Comment:
The same issue exists with `entriesCount` as with `lastEntry` (see other comment).